docker-compose.yml

```diff
@@ -16,14 +16,15 @@ services:
         environment:
             MARIADB_USER: root
             MARIADB_HOST: db
+        command: ["/scripts/wait-for", "db:3306", "--", "/scripts/pricing_import.py"]
```

Comment:
Maybe for consistency's sake this should go in `CMD` in `Dockerfile`, but I honestly don't know the difference.